src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/repository/QueryResult.kt

```diff
@@ -0,0 +1,21 @@
+package com.cognifide.gradle.aem.common.instance.service.repository
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+class QueryResult {
+
+    var success: Boolean = false
+
+    var results: Long = 0
+
+    var total: Long = 0
+
+    val more: Boolean = true
+
+    var offset: Long = 0
+
+    var hits: List<Map<String, Any>> = listOf()
```

Comment:
Why you have those props mutable? Query result looks like a good candidate to immutable object.

Reply:
yep, todo, thx